certif/translations/fr.json

```
@@ -10,7 +10,8 @@
       "delete": "Supprimer",
       "quit": "Quitter",
       "update": "Modifier",
-      "exit": "Quitter"
+      "exit": "Quitter",
+      "validate": "Validate"
```

Devrait être en français ?

```suggestion
      "validate": "Valider"
```

---

Bien vu !